README.md

```diff
@@ -599,6 +600,212 @@ cvvElement.mount('#cvv'); //Assumes there is a div with id='#cvv' in the webpage
   ]
 }
 ```
+## Using Skyflow Elements to update data
+
+You can update the data in a vault with Skyflow Elements. Use the following steps to securely update data. 
+
+### Step 1: Create a container
+Create a container for the form elements using the `container(Skyflow.ContainerType)` method of the Skyflow client:
+
+```javascript
+const container = skyflowClient.container(Skyflow.ContainerType.COLLECT)
+```
+
+### Step 2: Create a collect Element 
+Create a collect element. Collect Elements are defined as follows:
+
+```javascript
+const collectElement = {
+ table: "string", // Required, the table this data belongs to.
+ column: "string", // Required, the column into which this data should be updated.
+ type: Skyflow.ElementType, // Skyflow.ElementType enum.
+ inputStyles: {}, // Optional, styles that should be applied to the form element.
+ labelStyles: {}, // Optional, styles that will be applied to the label of the collect element.
+ errorTextStyles: {}, // Optional, styles that will be applied to the errorText of the collect element.
+ label: "string", // Optional, label for the form element.
+ placeholder: "string", // Optional, placeholder for the form element.
+ altText: "string", // (DEPRECATED) string that acts as an initial value for the collect element.
+ validations: [], // Optional, array of validation rules.
+ skyflowID: "string", // The skyflow_id of the record to be updated.
+};
+const options = {
+ required: false, // Optional, indicates whether the field is marked as required. Defaults to 'false'.
+ enableCardIcon: true, // Optional, indicates whether  the element needs a card icon (only applicable for CARD_NUMBER ElementType).
+ format: String, // Optional, format for the element (only applicable currently for EXPIRATION_DATE ElementType).
+ enableCopy: false, // Optional, enables the copy icon in collect and reveal elements to copy text to clipboard. Defaults to 'false').
+};
+const element = container.create(collectElement, options);
+```
+The `table` and `column1` fields indicate which table and column the Element corresponds to.
+
```

Comment:
Pls give valid column names, not as column1

Reply:
changes done